tools/db_bench_tool.cc

```diff
@@ -245,7 +245,15 @@ DEFINE_int32(threads, 1, "Number of concurrent threads to run.");
 DEFINE_int32(duration, 0, "Time in seconds for the random-ops tests to run."
              " When 0 then num & reads determine the test duration");
 
-DEFINE_int32(value_size, 100, "Size of each value");
+DEFINE_string(distribution_type, "fixed",
```

Comment:
`value_size_distribution_type` might be more clear for users.

Reply:
Thanks, I will update